packages/js-deployer/lib/builder/genesis-block.js

```diff
@@ -9,18 +15,22 @@ module.exports = class GenesisBlockBuilder {
    * @param  {Object} options
    * @return {void}
    */
-  constructor(network, options) {
+  constructor(network, options, config) {
```

Comment:
where does the config need to be passed from? I'm not seeing anything about it in the rest of the diff 🤔

Reply:
missed the `deployer` file as it has no extension and wasn't expanded